src/test/java/core/basesyntax/strategy/OperationHandlerBalanceTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.DataBase;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationHandlerBalanceTest {
+    private final HandlerConditionFactory handlerCondition = new HandlerConditionFactory();
+    private Fruit lemon;

```

Comment:
make this field local and use it only in test method, the same for other implementations test classes